src/mkfs.fat.c

```diff
@@ -1372,8 +1373,22 @@ int main(int argc, char **argv)
 	    program_name = p + 1;
     }
 
-    if (gettimeofday(&create_timeval, NULL) == 0 && create_timeval.tv_sec != (time_t)-1)
+    source_date_epoch = getenv("SOURCE_DATE_EPOCH");
+    if (source_date_epoch) {
+        char *tmp = NULL;
+        long long conversion = 0;
```

Comment:
You do not need to declare `tmp` and `conversion` again. It is on line 1347 and 1356.

Reply:
Oh, right! Fixed.